src/net/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ViewPanes } from '../constants'
+import { IEmbarkUser, IUser } from '../context'
+import browserHistory from '../history'
+import { makeRequest } from './request'
+
+const baseUrl = 'https://embark-prod.platform9.horse'
```

Comment:
This should be in a constants file

Reply:
This is going to change to be dynamic so I left it as is for now